wordCount.rb

```diff
@@ -0,0 +1,32 @@
+class WordCounter
+  attr_accessor :file, :output
+  def initialize(file)
+    @file = file
+    @output = {}
+    self.read
```

Comment:
Thanks for the feedback! I see what your getting at about chaining methods. I didn't think it was exceptionally clean and agree. On your comment regarding setting the default value of a hash to 0, what is the advantage of doing so? @paradime